src/ctapipe/io/tableloader.py

```diff
@@ -363,6 +364,12 @@ def read_observation_information(self):
         """
         return read_table(self.h5file, OBSERVATION_TABLE)
 
+    def read_scheduling_information(self):
```

Comment:
I know it's inconsistent with the `observation_info`, but I I really don't like `scheduling_information`, let's be specific and call it `read_scheduling_blocks`

Reply:
Done